content/docs/azure/OWNERS

```diff
@@ -0,0 +1,5 @@
+# Azure Docs Owners File
+approvers:  # No new approvers
+reviewers:
+  - berndverst
```

Comment:
@berndverst I think we should add you as a reviewer in the top-level OWNERS file for the kubeflow/website repository. Would you be happy with that solution? My thinking is that you're adding yourself as a reviewer, not an approver, which means that you're not acquiring the extra permissions that the approver role brings to the doc set as a whole. WDYT?

Reply:
Works for me. Let me do that instead!